src/EventListener/DeserializeListener.php

```diff
@@ -52,7 +52,10 @@ public function onKernelRequest(GetResponseEvent $event)
             || $request->isMethod('DELETE')
             || !($attributes = RequestAttributesExtractor::extractAttributes($request))
             || !$attributes['receive']
-            || ('' === ($requestContent = $request->getContent()) && $request->isMethod('PUT'))
+            || (
+                    '' === ($requestContent = $request->getContent())
+                    && (Request::METHOD_POST === $request->getMethod() || Request::METHOD_PUT === $request->getMethod())
```

Comment:
Can you store the result of `getMethod()` in a variable to avoid an unnecessary function call? As it's in a listener it can be performance sensitive.

Reply:
done, thanks for the feedback